src/main/java/seedu/address/model/person/NameAndSchoolContainsKeywordsPredicate.java

```diff
@@ -11,32 +12,70 @@
 public class NameAndSchoolContainsKeywordsPredicate implements Predicate<Person> {
```

Comment:
Probably need to change this class name eventually to accommodate searching by tag